src/KubernetesClient/KubernetesClientConfiguration.ConfigFile.cs

```diff
@@ -432,14 +427,12 @@ public static string ExecuteExternalCommand(ExternalExecution config)
 
             var process = new Process();
 
-            process.StartInfo.Environment.Add("KUBERNETES_EXEC_INFO",
-                JsonConvert.SerializeObject(execInfo));
-
+            process.StartInfo.EnvironmentVariables.Add("KUBERNETES_EXEC_INFO", JsonConvert.SerializeObject(execInfo));
             if (config.EnvironmentVariables != null)
             {
                 foreach (var configEnvironmentVariableKey in config.EnvironmentVariables.Keys)
                 {
-                    process.StartInfo.Environment.Add(key: configEnvironmentVariableKey,
+                    process.StartInfo.EnvironmentVariables.Add(key: configEnvironmentVariableKey,
```

Comment:
I checked the doc
EnvironmentVariables === Environment

just curious why this was changed

Reply:
If you are asking why I changed it, I had read above the target .net framework for this client was 4.5.2 which unfortunately doesn't have the Environment. However, EnvironmentVariables does exist in 4.5.2 and is in the latest preview of .net 5. [Source](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.processstartinfo.environmentvariables?view=netcore-3.1)

If you are asking why it changed in general, well, I can't help you there :) 

Hopefully that clears up any confusion.